File: 02_EstructurasControl/Repetitivas/For/CSharp/Calcular_ahorroSemanal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _1
{
    internal class Program
    {
        static void Main(string[] args)
        {
            // Calcula el ahorro semanal 
            double ahorro = 0;
            Console.WriteLine("¿Cuanto ahorras por dìa?");
            double dinero = Convert.ToDouble(Console.ReadLine());

            for (int i = 1; i <= 7; i++)
            {

                ahorro = ahorro + dinero;

            }
            Console.WriteLine("Tu ahorro total durante la semana es de: " + ahorro);
            Console.ReadKey (); 
        }
    }
}

File: 02_EstructurasControl/Repetitivas/For/Python/Calcular_ahorroSemanal.py
# Calcula el ahorro semanal

ahorro = 0.0
dinero = float(input("¿Cuánto ahorras por día? "))

for i in range(1, 8):  # de 1 a 7 (7 días)
    ahorro = ahorro + dinero

print("Tu ahorro total durante la semana es de:", ahorro)
